src/components/molecules/TableHeader/TableHeader.tsx

```diff
@@ -24,6 +19,7 @@ import { isTruthy } from "utils/types";
 import { Toggler } from "components/atoms/Toggler";
 
 import { EditTableTitleModal } from "./components/EditTableTitleModal";
+import { BackButton } from "components/atoms/BackButton";
```

Comment:
Import order sorting. With other `components/*`, not `./components/*`

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1492/commits/516bcef90616e538cbf0a8609a96d134bfbf4fcc